tracardi/process_engine/action/v1/converters/base64/encode/plugin.py

```diff
@@ -0,0 +1,78 @@
+from base64 import b64encode
+
+from tracardi.service.plugin.domain.register import Documentation, Form, FormComponent, FormField, FormGroup, \
+    MetaData, Plugin, PortDoc, Spec
+from tracardi.service.plugin.domain.result import Result
+from tracardi.service.plugin.runner import ActionRunner
+from .model.models import Configuration
+
+
+def validate(config: dict):
+    return Configuration(**config)
+
+
+class Base64EncodeAction(ActionRunner):
+    config: Configuration
+
+    async def set_up(self, init):
+        self.config = validate(init)
+
+    async def run(self, payload: dict, in_edge=None):
+        dot = self._get_dot_accessor(payload)
+        text = dot[self.config.to_base64]
+        result = b64encode(text.encode(self.config.input_encoding)).decode('ascii')
+
+        return Result(port="payload", value={"base64": result})
+
+
+def register() -> Plugin:
+    return Plugin(
+        start=False,
+        spec=Spec(
+            module=__name__,
+            className=Base64EncodeAction.__name__,
+            inputs=['payload'],
+            outputs=['payload'],
+            version='0.1',
+            license='MIT',
+            author='knittl',
+            manual='base64_encode',
+            init={
+                'to_base64': 'payload@data',
```

Comment:
As a default value I would use empty string. There may not be payload@base64

Reply:
No problem. Empty string means that the user has to always set it manually? Ok with me, just trying to understand the rationale (most plugins I looked at, generate `payload@data`, so I went with that).

Reply:
Empty is fine for this one.